File: QID-936-MVAnmdscar3/MVAnmdscar3.r

# clear all variables
rm(list = ls(all = TRUE))
graphics.off()

x = rbind(c(3, 2), c(2, 7), c(1, 3), c(0.37, 4.36), c(10, 4))

# plot
plot(x, pch = 19, col = "red", xlim = c(0, 12), ylim = c(0, 8), xlab = "", ylab = "", 
    main = "First iteration for Ferrari", cex.axis = 1.2, cex.lab = 1.2, cex.main = 1.8)
segments(x[3, 1], x[3, 2], x[4, 1], x[4, 2], lwd = 3, col = "red")
text(x, labels = c("Mercedes", "Jaguar", "Ferrari Init", "Ferrari New", "VW"), pos = 4) 

File: QID-936-MVAnmdscar3/MVAnmdscar3.m
%% clear all variables and console and close windows
clear
clc
close all

x = [3    2
    2    7
    1    3
    0.37 4.36
    10   4];

x = x';

%% plot
hold on
title('First Iteration for Ferrari')
xlim([0 12])
ylim([0 8])
labels = {'Mercedes    ', 'Jaguar      ', 'Ferrari Init',...
          'Ferrari New ', 'VW          '}
scatter(x(1, :), x(2, :), 90, 'r', 'fill')
text(x(1, :) + 0.2, x(2, :), labels)
line([x(1, 3) x(1, 4)], [x(2, 3) x(2, 4)], 'Color', 'r', 'LineWidth', 2)
box on
hold off
